tests/oclinter/bad-typedef/openconfig-testcase-succeed.yang

```diff
@@ -0,0 +1,37 @@
+module openconfig-testcase-succeed {
+  prefix "oc-tc";
+  namespace "http://openconfig.net/linter/testcase";
+
+  import openconfig-extensions { prefix oc-ext; }
+
+  description
+    "Success test case for using a bad type within
+    an OpenConfig mdoel.";
+
+  oc-ext:openconfig-version "0.0.1";
+
+  revision 2016-09-28 {
+    reference "0.0.1";
+    description
+      "Revision statement";
+  }
+
+  grouping testcase-config {
+    leaf bad {
```

Comment:
same comment as earlier if you want to make them consistent

Reply:
Done.